### DISCUSSION TITLE
failureDomain: chassis

### DISCUSSION THREAD

**Discussion by maxpain** [rook/rook#15101] [Q&A]
Hello. I have tree blade platforms, each has 8 blades (nodes), so 24 servers in total.
I want to set up 4+2 Erasure Coding, so we can survive a failure of one chassis.
I've added `topology.rook.io/chassis` labels to my nodes and tried to set `failureDomain: chassis` in pool settings, but the `pgs` aren't creating, they are stuck in the `creating+incomplete` state. I tried setting `min_size` to 4 instead of the default 5, but it didn't help.

Any ideas?

<img width="826" alt="image" src="https://github.com/user-attachments/assets/be051a27-3df6-473f-abad-d9eb11c19fef">

**Comment by travisn** [comment]
The failure domain must have at least the number of domains that the replication or erasure codings require. With 4+2 EC, you will require 6 chassis. But you only have 3 chassis, therefore the data cannot be spread across the expected failure domain. You might consider another level of the failure domain, such as two chassis in each rack:
```
rack a
  chassis s1
  chassis s2
rack b
  chassis s3
  chassis s4
rack c
  chassis s5
  chassis s6
```
This way, the 6 EC chunks can all be distributed across the 6 chassis.
